core/chains/evm/config/toml/defaults/BSC_Testnet.toml

```diff
@@ -0,0 +1,37 @@
+# BSC uses Clique consensus with ~3s block times
+# Clique offers finality within (N/2)+1 blocks where N is number of signers
```

Comment:
FYI for any readers - the `Clique` consensus is an example where the _difficulty_ field is still relevant and used to prioritize the signing order.

We should consider using the difficulty selection setting for the node pool, since a higher number head with a lesser total difficulty is possible and would be observed (unnecessarily) as a reorg if not tracking greatest total difficulty..

Reply:
https://smartcontract-it.atlassian.net/browse/BCF-1935

Reply:
Thanks for bringing this up. @simsonraj is going to dig a bit deeper into this.

Reply:
@davidcauchi , confirmed this is not a blocker now, we can go ahead as it is now.

Reply:
Thanks. Resolving